blackbox/sqllogictest/src/sqllogictest.py

```diff
@@ -36,8 +40,10 @@ def __init__(self, cmd):
         self.stmt = '\n'.join(cmd[1:])
 
     def execute(self, cursor):
+        stmt = RE_VARCHAR.sub('string', self.stmt)
+        stmt = RE_TEXT.sub('string', stmt)
```

Comment:
at the end you are doing the replace VARCHAR(d+) operation twice, or do i miss something?